docs/superpowers/plans/2026-05-04-rename-mac-to-nwg-and-release-040.md

```
@@ -0,0 +1,960 @@
+```bash
+git checkout main && git pull --ff-only
+git tag -a v0.4.0 -m "v0.4.0 — rename runtime artifacts to nwg-notifications-*"
+git push origin v0.4.0
+cargo publish
+gh release create v0.4.0 --title "v0.4.0" --notes "$(... see CHANGELOG ...)"
+```
+
+🤖 Generated with [Claude Code](https://claude.com/claude-code)
+EOF
+)"
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)